shared/reducers/teams.tsx

```diff
@@ -215,6 +215,19 @@ export default Container.makeReducer<
     const oldChannelInfo = mapGetEnsureValue(oldChannelInfos, conversationIDKey, Constants.initialChannelInfo)
     oldChannelInfo.memberStatus = RPCChatTypes.ConversationMemberStatus.left
   },
+  [TeamsGen.setChannelSelected]: (draftState, action) => {
+    const {teamID, channel, selected, clearAll} = action.payload
+    if (clearAll) {
+      draftState.selectedChannels.set(teamID, new Set())
+    } else {
+      const channelsSelected = mapGetEnsureValue(draftState.selectedChannels, teamID, new Set())
+      if (selected) {
+        channelsSelected.add(channel)
+      } else {
+        channelsSelected.delete(channel)
+      }
+    }
+  },
```

Comment:
@jakob223 why is this in the store rather than in component state? In the store it becomes effectively global state and makes the component not usable in other places.